gnat2goto/driver/tree_walk.adb

```diff
@@ -4814,8 +4814,40 @@ package body Tree_Walk is
             --  be called from Ada, or a foreign-language variable to be
             --  accessed from Ada. This would (probably) require gnat2goto to
             --  understand the foreign code, which we do not at the moment.
```

Comment:
FWIW this bit isn't actually true anymore; Since we now support linking of symtab files we _can_ actually do a multi language analysis just fine, we just have to make sure the name of the functions match.